README.md

```diff
@@ -86,7 +117,11 @@ Run the quickstart app on an Android device
 <img height="500px" src="images/quickstart/voice_activity.png">"
 
 
-Press the call button to connect to Twilio
+Press the call button to connect to Twilio. Enter a client name or a PSTN number in the call dialog. 
```

Comment:
> You can now dial Twilio to get an automated response. Leave the field empty and place the call. Dialing a client or phone number will be described in a later step.

Reply:
Add step 10 & 11 which describes how to dial a client or a number in more detail.

- Link to register your number
- Blurb about setting up 2 devices to dial each other